src/main/java/VASSAL/build/module/documentation/BrowserHelpFile.java

```diff
@@ -99,61 +101,64 @@ public void launch() {
   }
 
   /**
-   * The entry in the module Zip file containing the HTML directory
-   * @return
+   * @return The entry in the module Zip file containing the HTML directory
    */
   protected String getContentsResource() {
     return name == null ? null : name.replace(' ', '_');
   }
 
   protected void extractContents() {
-    ZipInputStream in = null;
-    try {
-      try {
-        in = new ZipInputStream(new BufferedInputStream(
-              GameModule.getGameModule().getDataArchive()
-                .getInputStream("help/" + getContentsResource()))); //$NON-NLS-1$
-      }
-      catch (IOException e) {
-        // The help file was created with empty contents.
-        // Assume an absolute URL as the starting page.
-        url = new URL(startingPage);
-        return;
```

Comment:
How are you handling this case now?

Reply:
Same way it was handled before, I moved this code to it's own method since it can be called from two catch blocks now:

```
  private void setFallbackUrl() {
    try {
      url = new URL(startingPage);
    }
    catch (MalformedURLException e) {
      logger.error("Malformed URL: {}", startingPage, e);
    }
  }
```